trulens_eval/examples/expositional/vector-dbs/pinecone/pinecone_evals_build_better_rags.ipynb

```diff
@@ -311,10 +311,11 @@
    "source": [
```

Comment:
<p>small nit: *each sentence is grounded by a retrieved context chunk.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/truera/trulens/pull/506/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
Fixed!